docs/GroupDefs.hpp

```diff
@@ -993,10 +991,18 @@ layer on top of Charm++.
 Custom reductions require one additional step to calling
 `contribute_to_reduction`, which is writing a reduction function to reduce the
 custom data. We provide a generic type that can be used in custom reductions,
-`Parallel::ReductionData`, which takes as template parameters the types it
-holds, similar to `std::tuple`. An example of a custom reduction function is:
-
-\snippet Test_AlgorithmReduction.cpp custom_reduce_function
+`Parallel::ReductionData`, which takes a series of `Parallel::ReductionDatum` as
+template parameters and `ReductionDatum::value_type`s as the arguments to the
+constructor. Each `ReductionDatum` takes up to four template parameters (two
+are required). The first is the type of data to reduce, and the second is a
```

Comment:
You could just refer to the documentation of ReductionDatum here instead of explaining the template parameters again. If you want do keep it like this, it would be good to format this enumeration as a list. Also a snippet would be very useful.

Reply:
I thought about that and preferred spelling it out here again for completeness and ease-of-reading, but I'm not so attached to it that if other people feel differently I'd argue. I'm happy to make it a list :)